searx/static/themes/simple/src/less/style-ltr.less

```diff
@@ -82,4 +82,8 @@
   transform: scale(1, 1);
 }
 
+.favicon {
+  margin: 0 8px 0 0;
+}
+
```

Comment:
When I wrote this, I was sure there must be a better way - but nothing I did worked, so I just kept it to discuss later. Can we use flex to maybe fix this?